### ISSUE TITLE
black-and-white accessibility

### ISSUE THREAD

**Issue by Psy-Fer** [Psy-Fer/kuva#14]
**What would you like?**
user on reddit
> Does it do cross hatching or image tiling? Plotting libraries always seem to forget this, but is basically required for black-and-white accessibility.

**Use case**
Accessibility

**Any prior art?**
find during research

**comments**

I should be able to do this using an alternate scene, then pushing that throught he backends with minimal changes to them

**Comment by Psy-Fer** [comment]
First stage of this is now done.

next stage is to add more control over patterns and introduce styling control and patterns more generally.
